readme.md

```diff
@@ -131,6 +131,13 @@ Default: [See above screenshot](https://github.com/yeoman/update-notifier#update
 
 Message that will be shown when an update is available.
 
+##### isGlobal
+
+Type: `boolean`<br>
+Default: `true`
+
+Include the `-g` argument in the default message's `npm i` recommendation. This is ignored if you supply your own `message`.
```

Comment:
I didn't want to make the description too long (as the rest were quite short), but if you feel like it's a good explanation, I'll work it in there. Thanks so much for all of this feedback!